src/components/Navbar/interfaces.tsx

```diff
@@ -24,7 +24,7 @@ export interface NavHeader extends NavItem {
 
 export interface NavLink extends NavItem {
   /** The link name */
-  label: string
+  label: React.ReactElement
```

Comment:
I think that we should be able to accept a string or label so that this is a non-breaking change.